sonic-net/sonic-buildimage
# [sonic-pit] Add PIT(Platform Integration Test) feature, second part, …

…add 6 test cases.

Signed-off-by: Li Hua <guizhao.lh@alibaba-inc.com>

#### Why I did it
sonic-pit is originated from S3IP(Simplify Switch System Integration Program) PIT(Platform Integration Test). It's designed to verify switch devices in the process of development, manufacturing and delivering stages. It focus on 2 roles: ODM and user. ODM may use PIT in developing and manufaturing stages, user may use PIT in delivery stage for acceptance test. It aims to provide an verify system for siwtch devices' hardware, drivers and firmware. More information, see HLD: https://github.com/clarklee-guizhao/SONiC/blob/pit/doc/pit/Platform_Integration_Test_high_level_design.md


#### How I did it
Porting sonic-pit from S3IP-PIT, make the original code clean enough, add PIT source as sonic-pit, add PIT running environment as docker-pit.
The whole PIT system is divided into several PR, this is the second PR with 6 test cases only.

#### How to verify it
First of all, this PR should be merged with the first PR(which has the PIT framework). After a sucessful build, copy sonic_pit-1.0-py3-none-any.whl to your device running sonic, install it with pip install sonic_pit-1.0-py3-none-any.whl, then run '/usr/local/pit-sysdiag/src/sysdiag -d' to start test.

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog

#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes

#### A picture of a cute animal (not mandatory but encouraged)
